Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/misc.yml

```diff
@@ -407,3 +407,21 @@
           Quantity: 5
         - ReagentId: Vitamin
           Quantity: 1
+         
+ - type: entity
```

Comment:
I believe you have 1 indentation too many here which means this doesn't work ingame.